### PR TITLE
Follow-up edits to PR#7297

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up edits to https://github.com/openshift/openshift-docs/pull/7297

@jmencak PTAL

### CODE REVIEW

File: scaling_performance/using_cluster_loader.adoc [hunk 1]
```diff
@@ -201,22 +201,25 @@ not successful.
...
 
 |`selectors`
-|A list of selectors to match pods in Running/Completed states.
+|A list of selectors to match pods in `Running` or `Completed` states.
 
 |`timeout`
-|A string. Synchronization timeout to wait for pods in Running/Completed states. For non 0 values use 
-units: [ns\|us\|ms\|s\|m\|h].
+|A string. The synchronization timeout period to wait for pods in `Running` or
+`Completed` states. For values that are not `0`, use  units:
```
Comment (jmencak) [inline scaling_performance/using_cluster_loader.adoc]: two spaces between "use" and "units"

File: scaling_performance/using_cluster_loader.adoc [hunk 2]
```diff
@@ -111,12 +111,12 @@ identical namespaces will be appended to `Basename` to prevent collisions.
...
 |`configmaps`
-|A list of key-value pairs. Key is ConfigMap name, value is a path to a file to create
-the ConfigMap from.
+|A list of key-value pairs. The key is the `ConfigMap` name and the value is a
+path to a file from which you create the `ConfigMap`.
 
 |`secrets`
-|A list of key-value pairs. Key is Secret name, value is a path to a file to create
-the Secret from.
+|A list of key-value pairs. The key is the secret name and the value is a path to
```
Comment (jmencak) [inline scaling_performance/using_cluster_loader.adoc]: Inconsistency.  "Secret" and "ConfigMap" are OpenShift objects.  Why have one definition quoted and one unquoted?  Not sure what the best approach is, OpenShift documentation has this inconsistency too.  Understand you could be talking about a "secret" in a more general sense, though.

**Review by jmencak — COMMENTED** [review]
LGTM on the whole, thank you!

**Comment by ahardin-rh** [comment]
@jmencak Thanks! Updated. 

Revision history is in the original PR.

**Pull request merged**